docs/editor/settings-sync.md

```diff
@@ -11,7 +11,7 @@ MetaDescription: Synchronize your user settings across all your Visual Studio Co
 
 Settings Sync lets you share your Visual Studio Code configurations such as settings, keybindings, and installed extensions across your machines so you are always working with your favorite setup.
 
->**Note**: Settings Sync is still in preview.
+>**Note**: Settings Sync is still in a 'preview'(alpha) stage-of-development.
```

Comment:
VS Code usually just says preview or stable and doesn't use terms like alpha or beta.